bee-network/bee-autopeering/src/peering/messages.rs

```diff
@@ -0,0 +1,176 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{local::salt::Salt, proto, request::Request};
+
+use base64 as bs64;
```

Comment:
Was it too long :trollface: ?

Reply:
It's [bs58](https://crates.io/crates/bs58)'s fault.....okay and also due to my perfidious need for consistency.

Reply:
I don't care if we use `base58` and `base64`, or `bs58` and `bs64`, but I feel like it should be consistent :smile: